### PR TITLE
File AddedQust.html

### PR DESCRIPTION
**Pull request by kalyani1942**

### CODE REVIEW

File: Kalyani/HTML/Assginment3/Qust1.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            ii.SYSTEM-2
+            <ul>
+                <li>
+                    Triveni
+                </li>
+                <li>Kalyani</li>
+            </ul>
+        </dd>
+    </dl>
+</body>
```
Comment (somesh6797) [inline Kalyani/HTML/Assginment3/Qust1.html:69]: add closing html tag here like < /html >

File: Kalyani/HTML/Assginment3/Qust1.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<head>
```
Comment (somesh6797) [inline Kalyani/HTML/Assginment3/Qust1.html:2]: add opening html tag here like <  html>

File: Kalyani/HTML/Assginment3/Qust1.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <title>
+        <h1>List/h1>
+    </title>
+</head>
+<body>
+    <h2>Ambersoft</h2>
+
+    <dl>
```
Comment (somesh6797) [inline Kalyani/HTML/Assginment3/Qust1.html:10]: at first there are two ordered list item so please use orderd list. not <   dd   > type of list 

for good understanding watch any nested list element video on youtube..

File: Kalyani/HTML/Assginment3/Qust1.html (hunk 1 above)
Comment (somesh6797) [reply Kalyani/HTML/Assginment3/Qust1.html:69]: /html

File: Kalyani/HTML/Assginment3/Qust1.html (hunk 3 above)
Comment (kalyani1942) [reply Kalyani/HTML/Assginment3/Qust1.html:10]: ohk

**Review by somesh6797 — CHANGES_REQUESTED** [review]
Changes required

**Review by sourabh-kanwade — APPROVED** [review]

File: Kalyani/HTML/Assginment3/Qust1.html (hunk 1 above)
Comment (kalyani1942) [reply Kalyani/HTML/Assginment3/Qust1.html:69]: Please review changes and approve it

**Review by somesh6797 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

**Pull request closed without merging**